client-v2/src/actions/Collections.ts

```diff
@@ -286,12 +304,100 @@ function getVisibleArticles(
   return fetchVisibleArticles(collectionType, articleDetails);
 }
 
+/**
```

Comment:
These have been moved to prevent a circular dependency -- arguably as they deal with collections in the state, not fronts, they're better placed here.